test/HorizontalMultivecTests.js

```diff
@@ -105,6 +106,35 @@ describe('Horizontal heatmaps', () => {
     hgc.instance().setState({ views });
   });
 
+  it('Test horizontal multivec with filtered rows', done => {
+    [div, hgc] = mountHGComponent(
+      div,
+      hgc,
+      horizontalMultivecWithFilteredRows,
+      () => {
+        const track = getTrackObjectFromHGC(
+          hgc.instance(),
+          'UiHlCoxRQ-aITBDi5j8b_w',
+          'YafcbvKDQvWoWRT1WrygPA'
+        ); // uuid of horizontal-multivec
+        const trackTiles = track.visibleAndFetchedTiles();
+        expect(trackTiles.length).toEqual(2);
+        expect(trackTiles[0].canvas.width).toEqual(256);
+        expect(trackTiles[0].canvas.height).toEqual(10);
+        expect(trackTiles[1].canvas.width).toEqual(256);
+        expect(trackTiles[1].canvas.height).toEqual(10);
```

Comment:
Thanks for the tests! They'd be even more effective if you created two tracks, one with the selectRows option and one without and checked `canvas.height`. That would provide a control.

Reply:
Definitely